dist/sgx/scripts/setup

```diff
@@ -4,28 +4,16 @@ pushd $(dirname $0)/.. > /dev/null
 ROOT_DIR="$(pwd)"
 popd > /dev/null
 
-# HSM binaries directory
-HSMBIN_DIR=$ROOT_DIR/hsm
-
-# HSM binaries
-HSMSGX_BINARY=hsmsgx
-HSMSGX_ENCLAVE_BINARY=hsmsgx_enclave.signed
+# HSM scripts directory
+SCRIPTS_DIR=$ROOT_DIR/scripts
 
 # Middleware binaries
 ADMIN_DIR="$ROOT_DIR/bin/adm_sgx"
 ADMIN_BUNDLE="$ADMIN_DIR.tgz"
-ADMIN="$ADMIN_DIR/adm_sgx"
-
-function cleanBinaries() {
-    rm -rf $ADMIN_DIR
-}
-
-function expandBinaries() {
-    cleanBinaries
+ADMIN_BIN="$ADMIN_DIR/adm_sgx"
 
-    mkdir -p $ADMIN_DIR
-    tar -xzmf $ADMIN_BUNDLE -C $ADMIN_DIR
-}
+# The admin command used to interact with the powhsmsgx service
+ADMIN_CMD="$ADMIN_BIN --host SGX"
```

Comment:
Shall we rename the host to `powhsmsgx`? I feel like `SGX` is both too generic and also uppercase which is not ideal.

Reply:
Sounds good, I can do that change